src/tickit/core/simulation.py

```diff
@@ -62,7 +62,7 @@ def build_simulation(
     backend: str = "internal",
     include_schedulers: bool = True,
     include_components: bool = True,
-    components_to_run: Optional[Set[ComponentID]] = None,
+    components_to_run: Optional[Union[Set[ComponentID], Literal["all"]]] = None,
```

Comment:
Could this stop being optional and default to "all" if they mean the same thing?

Reply:
That would have been the move yeah.